README.md

```diff
@@ -2,31 +2,818 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Define a query as an SQL SELECT statement that retrieves data from a database.
+- Use a comma-separated column list to display a subset of columns for a query.
+- Use the DISTINCT keyword to select unique values within a column.
+- Use the WHERE clause to select rows that match a predicate.
+- Use the IN keyword to retrieve rows that contain a value within a list of values.
+- Use the BETWEEN modifier to retrieve rows that contain a value that is within a given range.
+- Use the NULL data type keyword to retrieve rows containing a null value.
+- Use the ORDER BY modifier to order tables by specific SELECT statements.
+- Use the ASC and DESC modifiers to sort queries in ascending or descending order.
+- Use the LIMIT modifier to determine the number of rows to retrieve.
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+The SQL SELECT statement retrieves one or more table rows based on matching
+a predicate.  
 
-## Topic 1
+The general syntax for selecting rows from a single table is:
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+```roomsql
+SELECT [comma-separated list of columns]
+FROM [table_name]
+WHERE [row selection predicate]
+ORDER BY [comma-separated list of columns]
+LIMIT [count];
+```
 
-## Topic 2
+- The `WHERE` clause is optional.  The clause is used to select a subset of rows that match
+  the predicate.  If omitted, all rows are retrieved.
+- The `ORDER BY` clause is optional.  The clause is used to sort the result set based on
+  the order of one or more columns.  If omitted, the rows are not sorted and may appear in
+  any order.
+- The `LIMIT` clause is optional. The clause restricts the number of rows displayed in the result.
+
+
+## Table Setup  (Code Along)
+
+We will work with the same `pet` table from the previous lesson.  
+Copy and execute the following SQL statements in the**pgAdmin**
```

Comment:
Missing space between "the" and "**pgAdmin"**

Reply:
fixed